.ci_support/linux_64_python3.10.____cpython.yaml

```diff
@@ -6,3 +6,11 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.10.* *_cpython
+target_platform:
+- linux-64
```

Comment:
Please rever all changes to our hidden directories. Those should not be changed! If you need anything that implement changes in those, which I don't think is the case here, you will need to implement those via a local conda_build_config.yaml file.

Reply:
In that case, just rerender with latest conda-smithy.

Reply:
Done.